index.html

```diff
@@ -26,8 +25,159 @@ <h1>  Hey there. I’m Mariane <span class="green" >I’m a software developer</
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                 <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+            </div>
+            </section>
+            <article class="about2">
+                <h2>About me</h2>
+            <section class="about">
+               <div><img src="/images/illustration 1- about me-mobile.png" alt=""></div>
+            </section>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+            </article>
+            <div class="butt"><button class="btn" type="button"> Get My Resume</button></div>
+            <div class="groupimg"><img src="/images/Group.png" alt=""></div>
     
+            <article class="project2">
+               <div class="language-icon"> <h><img src="/images/icon-languages (1).png" alt=""></h></div>
+               <div class="lang-text">Languages </div>
+               <section class="">
+                <ul>
+                  <div class="lang-text-list">
+                    <li  class="li-lang">Javascript</li>
+                    <li class="li-lang">Ruby</li>
+                    <li class="li-lang">html</li>
+                    <li class="li-lang">css</li>
+                  </div>
+                </ul>
+               </section>
+
+            </article>
+            <article class="project2">
+                <div class="language-icon"> <h><img src="/images/icon-frameworks (1).png" alt=""></h></div>
+                <div class="lang-text">Frameworks </div>
+                <section class="">
+                 <ul>
+                   <div class="lang-text-list">
+                     <li class="li-lang">Bootstrap</li>
+                     <li class="li-lang">Ruby on rails</li>
+                     <li class="li-lang">Rspec</li>
+                     <li class="li-lang">css</li>
```

Comment:
Too many use of section and article tags from line 103 to 181. This makes your project very difficult to read and debug. You should consider the whole part of the about me as one section just as you did for the project part. You can check the image below for inspiration.

![Screenshot from 2022-09-23 13-22-32](https://user-images.githubusercontent.com/65233860/191970472-79c0267a-0baa-4350-b71e-b006fb9ec149.png)